guide_to_writing_techreports.tex

```diff
@@ -212,32 +214,30 @@ \subsection{Math}
 
 Notes:
 \begin{itemize}
-\item Scalar variables ($h$, $t$, $s$) are \emph{italicised} lower case.
+\item Scalar variables ($h$, $t$, $s$) are \emph{italicised} lower case (this includes parameters).
```

Comment:
Is the confusion here between variables and parameters a holdover from CPN? Should we perhaps just use the more generic "symbols" instead of variables?

Reply:
No, these students haven't had that yet, but I think that may work!